src/pages/About/AboutPage.jsx

```diff
@@ -0,0 +1,282 @@
+                <img src={RocketIcon} alt="" />
+              </div>
+            </div>
+            <div className={styles['welcome-graphics-row']}>
+              <Text className={styles['welcome-graphics-text']}>
+                Maybe you received a box from a friend.
+              </Text>
+              <Text className={styles['welcome-graphics-text']}>
+                Or found a box out in the world.
+              </Text>
+              <Text className={styles['welcome-graphics-text']}>
+                Or launched one for someone else to find.
+              </Text>
+            </div>
+          </div>
+          <div className={styles['welcome-graphics-container-mobile']}>
+            <div className={styles['welcome-graphics-row']}>
+              <div className={styles['welcome-graphics-icon']}>
+                <img src={FriendIcon} alt="" />
+              </div>
+              <Text className={styles['welcome-graphics-text']}>
+                Maybe you received a box from a friend.
+              </Text>
+            </div>
+            <div className={styles['welcome-graphics-row']}>
+              <div className={styles['welcome-graphics-icon']}>
+                <img src={GlobeIcon} alt="" />
+              </div>
+              <Text className={styles['welcome-graphics-text']}>
+                Or found a box out in the world.
+              </Text>
+            </div>
+            <div className={styles['welcome-graphics-row']}>
+              <div className={styles['welcome-graphics-icon']}>
+                <img src={RocketIcon} alt="" />
+              </div>
+              <Text className={styles['welcome-graphics-text']}>
+                Or launched one for someone else to find.
+              </Text>
+            </div>
+          </div>
+          <div>
+            <Text>
+              No matter how you found us, we&apos;re glad you did!
+              <br />
+              <br />
+              <strong>
+                This interactive map is a collection of all the FYA boxes in the world.
+              </strong>
+              <br />
+              <br />
+              We are strangers who care - worldwide.
+              <br />A diverse community, made stronger by our common goal to stop suicide.
+            </Text>
+          </div>
+        </div>
+        <div className={styles['site-description-section']}>
+          <Heading className={styles['about-heading']}>How This Works</Heading>
+          <Text className={styles['about-text']}>
+            Each box we launch into the world is added to this map. But it doesn&apos;t end there.
+            <br />
+            <br />
+            Find Your Anchor boxes are designed to be passed, or launched, from person to
+            person.&nbsp;
+            <strong>
+              Launching (and relaunching) boxes helps us reach more people, spread more love, and
+              grow this community.
```

Comment:
This should be on a new line on the mobile view.

Reply:
Fixed.